package/yast2-sap-ha.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-sap-ha
-Version:        4.5.6
+Version:        4.5.1
```

Comment:
It downgrades the version...

Reply:
The reason is, that no newer 4.5 version was ever relesed. This is the first one.